Makefile

```diff
@@ -28,8 +29,8 @@ update-dependencies: ## update golang dependencies
 
 generate:
 	$(GOGET) github.com/alvaroloes/enumer
+	$(GOGET) github.com/google/wire/cmd/wire
 	$(GOCMD) generate ./...
-	wire ./...
```

Comment:
@thomaspoignant Thanks for your return. I reverted my modification. Sorry🙏
Actually, we can update wire generated files with `go generate` but we can't generate them from scratch.

I think that's a  better approach to commit generated files. If you agree, we should create a new issue for that ;)

Reply:
I prefer to not commit generated files because the repo will be noisy and it avoid to use an old generated file from someone.

But we can open an issue and discuss about that. If we commit the generated file we should add something in the CI to validate that the generated file is the same if we re-generate the file.